gcp/api/README.md

```diff
@@ -27,3 +27,14 @@ https://cloud.google.com/container-registry/docs/advanced-authentication#gcloud-
 ## oss-vdb-test Project
 
 Automatically built and deployed on pushes to master branch.
+
+# Contributing:
+
+## Writing tests
+To speed up running tests:
+
+1. `make run-api-server` at the root of this repository to start the test server.
+2. `pipenv shell` in this directory
+3. `python -m unittest integration_test.py` in this directory.
```

Comment:
How hard is it to do this automatically as part of the `api-server-tests` make command?

Reply:
It is a decent amount of work, probably need a script that tests whether the server is running, if not build the docker image and run the server before running the test, otherwise run the test normally. Hmm... might be able to do a python script for it.